dss/storage/checkout/__init__.py

```diff
@@ -34,11 +34,18 @@ def validate_file_dst(replica: Replica, dst_bucket: str, dst_key: str):
 
 def pre_exec_validate(
         replica: Replica, dss_bucket: str, dst_bucket: str, bundle_uuid: str, bundle_version: str) -> bool:
+    validate_dst_name(dst_bucket)
     validate_dst_bucket(replica, dst_bucket)
     validate_bundle_exists(replica, dss_bucket, bundle_uuid, bundle_version)
     return True
 
 
+def validate_dst_name(dst_bucket: str)-> bool:
+    if len(dst_bucket.split(sep='/')) != 1:
```

Comment:
This is equivalent to `if "/" in dst_bucket:`

Reply:
changed to recommendation